kvdb-rocksdb/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+### Changed
+- Default column support removed from the API
+  - Column argument type changed from `Option<u32>` to `u32`
+  - Migration
+    - Column index `None` -> `0`, `Some(0)` -> `1`, `Some(1)` -> `2`, etc.
+    - Database should be opened with +1 number of columns for migration
```

Comment:
```suggestion
    - Database must be opened with at least one column
```

Reply:
I actually meant that if was previously was opened with `Some(9)`, then it should be opened with 10 columns with the new API to avoid migration.